data/desktop/io.snapcraft.SessionAgent.desktop.in

```diff
@@ -5,3 +5,4 @@ Exec=/bin/false
 DBusActivatable=true
 NoDisplay=true
 X-GNOME-UsesNotifications=true
+Icon=/usr/share/snapd/snapcraft-logo-bird.svg
```

Comment:
You could use
```suggestion
Icon=@icondir@/snapcraft-logo-bird.svg
```
And then add a `sed` command in the `%: %.in` rule in `data/desktop/Makefile`, similarly to how it's done for `bindir`.

Reply:
Done.